### PR TITLE
Add communications and outreach information

### PR DESCRIPTION
**Pull request by choldgraf**
This adds documentation about our communications and outreach channels, with the goal of making out strategy and channels of communication easier for others to understand and use.

- Defines the various accounts that we use and how to access them / request for access
- Defines some guidelines for what kinds of things to post
- Defines some checklists and steps to take to lower the decision-cost of posting
- Moves this out of the "Community Guidance" section into its own dedicated section.

closes https://github.com/2i2c-org/meta/issues/193

### CODE REVIEW

File: communication/blog.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+* **2i2c organizational updates**. Provide updates about important organizational decisions and changes. For example:
+    * Major changes to strategy
+    * Governance or other organizational changes
+    * New hires
+* **Major project updates**. For major projects and collaborations, we should provide regular updates about the work that we've done. This shouldn't frame 2i2c as the sole producer for these projects (unless we really are the sole producer), but demonstrate how 2i2c has contributed to the overall effort. For example:
+    * Major deliverables that we’ve met on a project
+    * New collaborations
+* **Open source updates**. If we do particular service work, or make a strong technical improvement in an open source project, we should tell the world about it. 
+* **Updates from the hubs pilot**. As our pilot hub infrastructure grows and evolves, and as we serve more communities with it, we should tell others about the impact we are having.
```
Comment (damianavila) [inline communication/blog.md:42]: Maybe we should not keep using the pilot hub concept? I think we have removed it from other places already...

**Comment by choldgraf** [comment]
I believe that I've addressed all of your questions @damianavila ! I've also added "how to access" information for each of the accounts there (and if you / others would like access please just let me know!)

**Review by damianavila — APPROVED** [review]

**Comment by damianavila** [comment]
> I've also added "how to access" information for each of the accounts there 

Super, thanks!

> (and if you / others would like access please just let me know!)

Yes, I would like to get access to them (to further ANN the job posting).

**Comment by choldgraf** [comment]
FYI - I've:

- Added @damianavila as a "Super Admin" on our LinkedIn.
- Given him the password to our Twitter account

**Comment by choldgraf** [comment]
ok will merge this in as well, since I think it mostly just makes a bunch of information available that was not available before, rather than defining new policy!

**Comment by choldgraf** [comment]
@damianavila yep, see Slack

**Pull request merged**